cms/djangoapps/contentstore/helpers.py

```diff
@@ -210,6 +213,14 @@ def create_definition(self, block_type, slug=None) -> DefinitionLocator:
         return DefinitionLocator(block_type, LocalId(block_type))
 
 
+@frozen
+class StaticFileNotices:
+    """ Information about what static files were updated (or not) when pasting content into another course """
+    new_files: list[str] = Factory(list)
+    conflicting_files: list[str] = Factory(list)
+    error_files: list[str] = Factory(list)
+
+
 def import_staged_content_from_user_clipboard(parent_key: UsageKey, request):
```

Comment:
```suggestion
def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) -> tuple[XBlock | None, StaticFileNotices]:
```

If I'm not mistaken, this is the current return type. We can add it, as it's not obvious.